src/org/rascalmpl/util/maven/Dependency.java

```diff
@@ -87,7 +89,13 @@ static Dependency build(org.apache.maven.model.Dependency d, IListWriter message
             case "provided": scope = Scope.PROVIDED; break;
             case "runtime": scope = Scope.RUNTIME; break;
             case "test": scope = Scope.TEST; break;
-            case "system": scope = Scope.SYSTEM; break;
+            case "system":
+                scope = Scope.SYSTEM;
+                if (d.getSystemPath() == null) {
```

Comment:
this is different than all the other cases. Normally a dependency is not resolved until we request it.

This is to make sure we can report errors to the user, and not miss them.

Reply:
Changed this approach to let Maven do its thing and just not crash when `systemPath` is null.